### DISCUSSION TITLE
Welcome to phpbrew Discussions!

### DISCUSSION THREAD

**Discussion by c9s** [phpbrew/phpbrew#1286] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peter279k** [comment]
Hi 👋, I'm Peter Li and I do some following works in the project 👇 👇:

- Review the PR.
- Making some released versions.
- Write official documentations.
- Maintain Wiki.
- Issue resolver.
- The collaborator.
